front/pages/w/[wId]/members/index.tsx

```diff
@@ -129,343 +392,257 @@ export default function WorkspaceAdmin({
     if (!res.ok) {
       window.alert("Failed to invite new member to workspace.");
     } else {
+      setSuccessMessage(
+        `Invite sent to ${inviteEmail}. You can repeat the operation to invite other users.`
+      );
       await mutate(`/api/w/${owner.sId}/invitations`);
     }
     setIsSending(false);
     setInviteEmail("");
-  };
+  }
+}
 
-  const handleRevokeInvitation = async (invitationId: number) => {
-    const res = await fetch(`/api/w/${owner.sId}/invitations/${invitationId}`, {
+function InviteSettingsModal({
+  showModal,
+  onClose,
+  owner,
+}: {
+  showModal: boolean;
+  onClose: () => void;
+  owner: WorkspaceType;
+}) {
+  const [domainUpdating, setDomainUpdating] = useState(false);
```

Comment:
NIT: set the type for all your `useState`?

`  const [domainUpdating, setDomainUpdating] = useState<boolean>(false);`

Reply:
Here I would argue that it's fine not to set the type for native types (boolean, string, number) since it's unambiguous from the set value so you know it when you see the type